openedx/core/djangoapps/content/search/tests/test_api.py

```diff
@@ -479,7 +479,7 @@ def test_init_meilisearch_index(self, mock_meilisearch) -> None:
     def test_reindex_meilisearch_collection_error(self, mock_meilisearch) -> None:
 
         mock_logger = Mock()
```

Comment:
You can delete this mock and the related assertion below if we're not using it anymore. Same in the next test.